examples/AzureSDKSmokeTest/AzureSDKSmokeTest.xcodeproj/project.pbxproj

```diff
@@ -360,6 +360,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				DEVELOPMENT_TEAM = UBF8T346G9;
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```

Comment:
Why does this need to be preserved?

Reply:
Otherwise it will fall back to excluding arm64 simulators and `AzureSDKSmokeTest.xcworkspace` won't be run on simulators for M1.

For `AzureSDK.xcworkspace` projects I was able to remove the excluded arch from the project and the targets but  the project in `AzureSDKSmokeTest.xcworkspace` already didn't include the excluded arch setting, but the target did.